### PR TITLE
XW-1690 | support breakpoints in global navigation

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/XW-1690
https://github.com/Wikia/design-system/pull/44

@Wikia/x-wing

### CODE REVIEW

**Comment by kvas-damian** [comment]
Please resolve merge conflicts

File: gulpfile.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ gulp.task('build-styles', ['clean-up'], function () {
 });
 
 gulp.task('build-html', ['build-styles'], function () {
-	return gulp.src(['./styles/*.scss', './components/*.md'])
+	return gulp.src('./components/global-navigation.md')
```
Comment (kvas-damian) [inline gulpfile.js]: Please revert this change

**Comment by kvas-damian** [comment]
LGTM

File: styles/_breakpoints.scss [hunk 2]
```diff
@@ -40,6 +42,8 @@
 	tablet-only: $breakpoint-tablet-only,
 	desktop-up: $breakpoint-desktop-up,
 	desktop-only: $breakpoint-desktop-only,
+	desktop-l-up: $breakpoint-desktop-l-up,
```
Comment (hakubo) [inline styles/_breakpoints.scss]: what is l?
large?

how about desktop-large-*?

**Comment by hakubo** [comment]
one minor comment, lgtm
